inc/Compatibility/Compatibility.php

```diff
@@ -70,13 +70,30 @@ public static function get_supported_post_types(): array {
 	}
 
 	/**
-	 * Check if the Gutenberg plugin is active.
-	 *
-	 * TODO: Check GUTENBERG_VERSION in production to ensure it is running a
-	 * compatible version.
+	 * Check if the Gutenberg plugin is active, and meets the minimum version requirement.
 	 */
 	private static function is_gutenberg_plugin_active(): bool {
-		return defined( 'IS_GUTENBERG_PLUGIN' ) && constant( 'IS_GUTENBERG_PLUGIN' );
+		// Gutenberg plugin isn't active.
+		if ( ! defined( 'IS_GUTENBERG_PLUGIN' ) || ! constant( 'IS_GUTENBERG_PLUGIN' ) ) {
+			return false;
+		}
+
+		// Need to ensure that development mode bypasses this check.
+		// For dev builds, this is defined in the Gutenberg plugin's main file.
+		// For production builds, this is removed entirely.
+		if ( defined( 'GUTENBERG_DEVELOPMENT_MODE' ) && constant( 'GUTENBERG_DEVELOPMENT_MODE' ) ) {
+			return true;
+		}
+
+		// If its not development mode, and the version is set, check the version.
+		if ( defined( 'GUTENBERG_VERSION' ) && is_string( constant( 'GUTENBERG_VERSION' ) ) ) {
+			/** @var string $gutenberg_version */
+			$gutenberg_version = constant( 'GUTENBERG_VERSION' );
+			return version_compare( $gutenberg_version, '21.7.0', '>=' );
```

Comment:
Why does this compare against 21.7.0 while the copy says 21.9.0+? Should they both pull from a constant?

Reply:
I'm going to provide a constant at the top, rather than one like the websocket url that can be overriden. I think the constant in the code should be fixed, as it means that GB version is tied to that code. It's not exactly accurate since there's some changes in our wpvip/rtc-plugin but still.